src/App.jsx

```diff
@@ -14,70 +16,72 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [selectedGood, setSelectedGood] = useState('Jam');
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
+  const clear = () => setSelectedGood('');
 
-      <button
-        data-cy="ClearButton"
-        type="button"
-        className="delete ml-3"
-      />
-    </h1>
+  return (
+    <main className="section container">
+      {selectedGood ? (
+        <h1 className="title is-flex is-align-items-center">
+          {`${selectedGood} is selected`}
+          <button
+            onClick={() => clear()}
+            data-cy="ClearButton"
+            type="button"
+            className="delete ml-3"
+          />
+        </h1>
+      ) : (
+        <h1 className="title is-flex is-align-items-center">
+          No goods selected
+        </h1>
+      )}
 
-    <table className="table">
-      <tbody>
-        <tr data-cy="Good">
-          <td>
-            <button
-              data-cy="AddButton"
-              type="button"
-              className="button"
-            >
-              +
-            </button>
-          </td>
+      <table className="table">
+        <tbody>
+          {goods.map((good) => {
+            const isSelected = selectedGood === good;
 
-          <td data-cy="GoodTitle" className="is-vcentered">
-            Dumplings
-          </td>
-        </tr>
+            return (
+              <tr
+                key={good}
+                data-cy="Good"
+                className={classNames(
+                  isSelected ? 'has-background-success-light' : '',
```

Comment:
```suggestion
                  'has-background-success-light': isSelected,
```

Reply:
I would like to know more about how I can implement it

Reply:
When i paste code from your recomendation i recive error. Pls check my screenshot
<img width="1039" alt="Screenshot at Nov 24 20-33-11" src="https://github.com/mate-academy/react_goods-selector-js/assets/137291152/0b830ba1-60d8-4cff-919a-7b5c5d2d0638">

Reply:
This expression must be in curly brackets
<img width="487" alt="image" src="https://github.com/mate-academy/react_goods-selector-js/assets/64750958/70b49445-fe4c-4b26-b1cd-8af10620d328">